casper/src/test/scala/coop/rchain/casper/EstimatorHelperTest.scala

```diff
@@ -110,24 +110,6 @@ class EstimatorHelperTest
       }
   }
 
-  it should "conflict if their deploys conflict in the paymentLog" in withStorage {
```

Comment:
a thought: once we introduce system deploys, we need to remember to consider them when determining conflicts. I'll add a test for that in the gist.